.vscode/settings.json

```diff
@@ -6,5 +6,14 @@
     "editor.formatOnSave": true, // optional 
     "editor.formatOnSaveMode": "file", // required to format on save
     "files.autoSave": "onFocusChange", // optional but recommended
-    "vs-code-prettier-eslint.prettierLast": false // set as "true" to run 'prettier' last not first
```

Comment:
?

Reply:
Running the prettier first will prevent the line endings to stop being a pain in the ass (at least for me), I can reverse it if you want